Runtime/CesiumCameraManager.cs

```diff
@@ -0,0 +1,107 @@
+using Reinterop;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using UnityEngine;
+
+namespace CesiumForUnity
+{
+    /// <summary>
+    /// Manages the set of cameras that are used for Cesium3DTileset culling and level-of-detail.
+    /// </summary>
+    [DisallowMultipleComponent]
+    [AddComponentMenu("Cesium/Cesium Camera Manager")]
+    [Icon("Packages/com.cesium.unity/Editor/Resources/Cesium-24x24.png")]
+    public class CesiumCameraManager : MonoBehaviour
+    {
+        /// <summary>
+        /// Gets the instance suitable for use with the given game object.
+        /// </summary>
+        /// <remarks>
+        /// This method looks for an existing instance using `GetComponentInParent`. If it fails to find one, then
+        /// it will create one. When one is created, it be added to the same `GameObject` that has the
+        /// `CesiumGeoreference` (found using `GetComponentInParent` again) if there is one. If there is no
+        /// `CesiumGeoreference`, the instance is added to the originally-provided `GameObject`.
+        /// </remarks>
+        /// <param name="gameObject">The game object.</param>
+        /// <returns></returns>
+        public static CesiumCameraManager GetOrCreate(GameObject gameObject)
+        {
+            if (gameObject == null) throw new ArgumentNullException("gameObject");
+
+            CesiumCameraManager result = gameObject.GetComponentInParent<CesiumCameraManager>();
+            if (result == null)
+                return CesiumCameraManager.Create(gameObject);
+            else
+                return result;
+        }
+
+        public static CesiumCameraManager GetOrCreate(Component component)
```

Comment:
Is this function meant to be `public`? If so, maybe mirror the documentation from the other `GetOrCreate` overload

Reply:
I made it internal (it can't be private because it's called from C++). I considered just removing it, but it's useful as a tiny optimization to avoid crossing between C++ / C# twice when we need to get the camera manager.

Reply:
Actually, I changed my mind, I'm going to remove it. It's only called in `LoadTileset`, not every frame, so it's an irrelevant optimization. Less code is better.